### PR TITLE
More types supported

### PR DESCRIPTION
**Pull request by Raalsky**

### CODE REVIEW

**Review by AleksanderWWW — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -293,6 +293,79 @@ __Returns__:
...
+#### `fetch()` or `fetch_last()`
+Retrieves last series value either from the internal cache (see [`prefetch()`](#prefetch)) or from the API.
+
+__Example__:
+```python
+loss = run["loss"].fetch_last()
+```
+
+__Returns__:
+`Optional[float]`
```
Comment (normandy7) [inline README.md:348]: Intentional?

**Review by normandy7 — APPROVED** [review]
